.github/workflows/pulsar-ci.yaml

```diff
@@ -953,11 +954,11 @@ jobs:
   # It cleans up the binaries in the same job in order to not spin up another runner for basically doing nothing.
   pulsar-ci-checks-completed:
     name: "Pulsar CI checks completed"
-    if: always()
+    if: ${{ !cancelled() }}
```

Comment:
I'm not sure. If I cancel the workflow manually then this job is marked as skipped and then GH allows me merge the pull. Could you verify it ?